app/views/spendings/index.html.erb

```diff
@@ -0,0 +1,37 @@
+<div>
+  <div>
+  <% @header = "Transactions" %>
+	<%= render "shared/header", locals: { header: @header }, back: true %>
+  <div class="mb-10">
+  <% if @spendings.empty?%>
+    <p class="text-center pt-16 text-xl">You don't have any transactions yet !!</p>
+  <% else %>
+    <ul class="flex flex-col justify-center pt-12 ">
+      <li class="w-[90%] mx-auto flex flex-col items-center gap-3 shadow-lg px-3 py-5  my-5 bg-white">
+          <div class="flex  justify-center items-center">
+              <div class="text-4xl me-4">
+                <%= image_tag(@group.icon, alt: "Group Icon", class: "img-fluid", style: "height: 100px; width:100px;") %>
+              </div>
```

Comment:
As requested by the previous code reviewer kindly make sure your transaction page matches the given wireframe design. When I add two transactions to a given category the create new transaction button disappears.

This is how your transaction page looks:

![Screenshot 2023-11-25 at 7 26 16 PM](https://github.com/arnoldnekemiah/Budget-app/assets/79506105/95cd37a8-6e2f-48f3-b864-71c18475a46e)


Your transaction page should look like this:
![Screenshot 2023-11-25 at 7 34 39 PM](https://github.com/arnoldnekemiah/Budget-app/assets/79506105/8828c2fd-b831-4a98-af07-2eee1e999bdc)

Kindly pay attention to detail